modules/ROOT/pages/configure-horizontal-autoscaling.adoc

```diff
@@ -0,0 +1,240 @@
+
+In Kubernetes, a Horizontal Pod Autoscaler (HPA) automatically updates a workload resource, with the aim of automatically scaling the workload to match demand. Horizontal scaling means that the response to increased load is to deploy more pods.
+
+[NOTE]
+HPA is supported only on Runtime Fabric agent version 2.6.22 and higher.
+
+This feature applies only to select customers who are opted into the new pricing and packaging model. For more details, see xref:general::pricing.adoc[]
+
+
+== Before You Begin
+
+To configure autoscaling, the cluster administrator must enable the https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#support-for-metrics-apis[support for metrics APIs^] on your managed K8s API server.
+Refer to your managed vendors documentation on how to install or enable metrics API servers.
+
+
+== Recommendations for Cluster Autoscaling
+
+The following recommendations are for your infrastructure team to configure cluster autoscaling
+
+=== Configure Node Groups
+
+Configure one node group per availability zone, with a minimum of one node in the node group.
+This ensures that Kubernetes has information about all the availability zones in the cluster and avoids an issue where zone topology unschedulable pod fails to scale-out topologySpread predicate check.
+
+Make sure that nodes from all the node groups have the exact same labels except for the common labels and specific cloud provider labels that are different across nodes or availability zones by default.
+
+Custom labels (which are not common labels and specific cloud provider labels) and are different across nodes need to be excluded from cluster autoscaler when evaluating similar node groups.
+
+For reference on managed EKS labels such as:
+----
+eks.amazonaws.com/nodegroup
+failure-domain.beta.kubernetes.io/region
+failure-domain.beta.kubernetes.io/zone
+kubernetes.io/hostname
+topology.kubernetes.io/region
+topology.kubernetes.io/zone
+----
+
+
+=== Configure Cluster Autoscaler Flags
+
+Start the cluster autoscaler with flag `--balance-similar-node-groups=true` and exclude the custom labels previously mentioned in the node groups section with the flag `--balancing-ignore-label`.
+These flags ensure that the cluster autoscaler balances the number of nodes across availability zones. 
+
+For reference on managed EKS cluster, the following node labels are different across node groups:
+
+----
+k8s.amazonaws.com/eniConfigCustom
+vpc.amazonaws.com/eniConfig
+----
+
+Therefore, the cluster autoscaler starts with:
+
+----
+- --balancing-ignore-label=k8s.amazonaws.com/eniConfigCustom
+- --balancing-ignore-label=vpc.amazonaws.com/eniConfig
+----
+
+=== Configure Overprovisioning Pod
```

Comment:
```suggestion
=== Configure Overprovisioning Node
```
I think this should be over provisioning nodes, not pods.

Also need to apply the corresponding change in the text.